README.md

```diff
@@ -80,6 +80,7 @@ cd [WHERE_YOU_CLONED_AIRPLAY2_RECEIVER]
 virtualenv airplay2-receiver
```

Comment:
I'd like to see this changed to something a little more obvious than the name of the project itself.
Personally, I went for `ap2env`, however if you feel that something else is more appropriate go ahead.

I'd just like to see it renamed from `airplay2-receiver` to almost anything else, preferrably including `env`.